docs/source/PaaSTA_Development.rst

```diff
@@ -0,0 +1,108 @@
+==================
+PaaSTA Development
+==================
+
+
+What is PaaSTA playground
+-------------------------
+
+PaaSTA playground allows the developer to run and debug paasta locally on an ephemeral Kubernetes cluster. Currently, it includes the following components:
+generating ``etc_paasta_playground`` and ``soa_config_playground`` directories, setup_kubernetes_job (deploy compute-infra-test-service),
+PaaSTA API, and PaaSTA CLI client.
+
+Create a Kind Cluster - K8s Cluster
+-----------------------------------
+
+Before running or debugging PaaSTA playground, you will need to create a kubernetes cluster - Kind. You can do so by running the make target
+``make create_cluster`` from ``paasta/k8s_itests`` directory.
+
+To delete the cluster, run ``make clean`` from ``paasta/k8s_itests`` directory.
+
+Debugging PaaSTA playground (in VS Code)
+----------------------------------------
+
+All components of PaaSTA playground are easy to debug in VS Code by running the debugging configurations in ``launch.json`` file. Before you start
+debugging, first run ``make vscode_settings``. This make target will add debugging settings in ``launch.json`` file.
+
+Below outlines the steps to run the debugger in VS Code and to debug PaaSTA playground components.
+
+.. _running-debugger:
+
+Running debugger (in VS Code)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In order to run the debugger with any of the configurations in ``launch.json``, you can go to ``Run and Debug`` tab in VS Code
+
+.. image:: run_and_debug_tab.png
+   :width: 700px
```

Comment:
nit: could we make these images 700px wide so that we don't need to resize them in the browser? should also make the file size smaller if we shrink the picture size :)

Reply:
ohh sorry I'm confused. The images width is already set to 700px, do you mean to decrease this value?

Reply:
oh sorry, i meant: we should make the raw image be exactly the size that we want to display it as rather than having browsers/rst resize the image for us (i.e., edit the image to be exactly 700px wide)